manifests/release/olm/bundle/kubevirt.VERSION.csv.yaml.in

```diff
@@ -0,0 +1,111 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kubevirtoperator.{{.CsvVersion}}
+  namespace: placeholder
+  annotations:
+    capabilities: "Full Lifecycle"
+    categories: "Virtualization"
+    alm-examples: |
+      [
+        {
+          "apiVersion":"kubevirt.io/v1alpha3",
+          "kind":"KubeVirt",
+          "metadata": {
+            "name":"kubevirt",
+            "namespace":"kubevirt"
+          },
+          "spec": {
+            "imagePullPolicy":"Always"
+          }
+        }
+      ]
+    description: Creates and maintains KubeVirt deployments
+spec:
+  displayName: KubeVirt
+  description: |
+    KubeVirt is a cluster add-on that provides a way to run virtualized workloads alongside containerized workload in a Kubernetes / OpenShift native way.
+
+    _The KubeVirt Operator does not support updates yet._
+  keywords:
+    - KubeVirt
+    - Virtualization
+  version: {{.CsvVersion}}
+  maturity: alpha
```

Comment:
Not sure what impact this would have but I noticed that package refers to a `beta` channel but this says the CSV is `alpha`

Reply:
not sure if those fields are related, will check, thx!